docs/topics_5/web-openshift-deploy.adoc

```diff
@@ -1,19 +1,34 @@
+<<<<<<< HEAD
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='deploy_mta_app_openshift_{context}']
 = Deploy the {ProductShortName} Application
+=======
+[[deploy_mta_app_openshift]]
+= Deploying the {ProductShortName} application
+>>>>>>> minor
```

Comment:
Looks like you didn't complete fix the merge conflict. You need to remove the >>> minor